uwb-beacon-firmware/STM32F405xG.ld

```diff
@@ -88,4 +88,7 @@ REGION_ALIAS("HEAP_RAM", ram0);
 _config_start = ORIGIN(config);
 _config_end = ORIGIN(config) + LENGTH(config);
 
+__process_stack_size__ = 0x400;
+__main_stack_size__ = 0x400;
```

Comment:
This will be replaced by a linker flag? Isn't this already done in `ChibiOS_LD.cmake`?

Reply:
For now its not a linker flag, I am not sure we are using `ChibiOS_LD.cmake`. I actually prefer having it in the linker script rather than in a flag, it feels cleaner and easier to review.

Reply:
Ok, agree. It's better to define such parameters in the source than in build system files.